### PR TITLE
doc: fix some typos in Error Handling doc

### PR DESCRIPTION
**Pull request by FabioPinheiro**

### CODE REVIEW

**Review by antonbaliasnikov — APPROVED** [review]

File: Mediator-Error_Handling.md [hunk 1]
```diff
@@ -31,28 +31,28 @@ This table defines the expected the behavior of the mediator in different scenar
...
   - S1D - Reply with the exact same message (encrypted with the same ephemeral keys)
 
-- **G2** - Receive a message for a unsupported protocol
+- **G2** - Receive a message for an unsupported protocol
   https://input-output.atlassian.net/browse/ATL-5840
   - G2A - ignore (no effect/output) 
   - G2B - Send a problem report "e.p.msg.unsupported"
 
-- **G3** - Receive a message for a unsupported version of the protocol.
+- **G3** - Receive a message for an unsupported  unsupported version of the protocol.
```
Comment (petevielhaber) [inline Mediator-Error_Handling.md:47]: - Remove the second "unsupported"

File: Mediator-Error_Handling.md [hunk 2]
```diff
@@ -31,28 +31,28 @@ This table defines the expected the behavior of the mediator in different scenar
...
   - G2B - Fallback to G2
   - G3B - Send `e.p.msg.unsupported` and say what version(s) its supported
 
 - **G4** - When an internal error or any unexpected error happens.
   - G4A - ignore (no effect/output) 
   - G4B - Send a problem report "e.p.error"
 
-- **G5** - If the message is tampered (got crypto errors).
+- **G5** - If the message is tampered (or it throws any crypto errors).
```
Comment (petevielhaber) [inline Mediator-Error_Handling.md:55]: - **G5** - If the message is tampered with (or throws any crypto errors).

File: Mediator-Error_Handling.md [hunk 3]
```diff
@@ -66,8 +66,8 @@ This table defines the expected the behavior of the mediator in different scenar
 
 - **G8** - When a parsing error from the decrypted message.
   - G8A - ignore (no effect/output)
-  - G8B - (sync!) Send a `e.p.msg` If the plaintext message is malformed 
-  - G8C - Send a `e.p.msg.<PIURI>` If the plaintext message is malformed or if parsing into a specific protocol's data model fails.
+  - G8B - (sync!) Send an `e.p.msg` If the plaintext message is malformed 
+  - G8C - Send an `e.p.msg.<PIURI>` If the plaintext message is malformed or if parsing into a specific protocol's data model fails.
```
Comment (petevielhaber) [inline Mediator-Error_Handling.md:70]: - G8C - Send an `e.p.msg.<PIURI>` If the plaintext message is malformed or parsing into a specific protocol's data model fails.

**Review by petevielhaber — COMMENTED** [review]
- Minor change requests.
- Note: Fabio not sure why but it wasn't let me make suggestions. I did add the full text for 2 suggestions for copy/paste.

File: Mediator-Error_Handling.md (hunk 1 above)
Comment (FabioPinheiro) [reply Mediator-Error_Handling.md:47]: Almost lost your review if not for the email.
I will have a new PR with it

**Comment by FabioPinheiro** [comment]
The minor change requests will be in https://github.com/input-output-hk/atala-prism-mediator/pull/137

**Pull request merged**